website/content/docs/enterprise/lts.mdx

```diff
@@ -0,0 +1,192 @@
+      <th style={{verticalAlign: 'middle'}}>Answer</th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#who">Who should consider long term support?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      Enterprise customers using Vault for sensitive or critical workflows.
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#what">What is long-term support?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      Extended support for select Vault versions. HashiCorp supports LTS
+      versions for two years. In the first year, LTS Vault versions receive
+      feature and critical updates. In the second year, LTS Vault versions
+      continue receiving critical updates.
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#where">Where do I download the LTS binary?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      You do not need to download a separate binary for LTS support. Just make
+      sure to select an LTS Vault version when you <a href="/vault/install">install</a>
+      or <a href="/vault/docs/upgrading">upgrade</a> your Vault instance.
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#when">When are LTS versions released?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      As of Vault Enterprise 1.16, the first release of a calendar year includes
+      long-term support.
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#why">Why should I avoid upgrading from an LTS version to a non-LTS version?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      If you upgrade to a non-LTS version, your Vault instance will revert to
+      a standard support version, which will stop receiving critical updates
+      when it leaves the standard support window.
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <a href="#how">How do I update an LTS Vault installation?</a>
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      Follow your existing Vault upgrade process, but allow extra time for the
+      possibility of transitional upgrades across multiple Vault versions.
+    </td>
+  </tr>
+</tbody>
+</table>
+
+<a id="who" />
+
+## Who should consider long term support?
+
+Vault upgrades are challenging, especially for sensitive or critical workflows,
+extensive integrations, and large-scale deployments. Strict upgrade policies
+also require significant planning, testing, and employee hours to execute
+successfully. 
+
+Customers who need assurances that their current installation will receive
+critical bug fixes and security patches with minimal service disruptions should
+consider moving to a Vault version with long-term support.
+
+<a id="what" />
+
+## What is long-term support?
```

Comment:
***If*** you can add a visual diagram like the one in the [PRD](https://docs.google.com/document/d/1XOPN8Ba6Dcqz6gW_g212LuVMXxjGAuJSxSvFvA9gqXA/edit?usp=sharing), that may help even more?  At least for me, the diagram helped me understand the overall picture.  But that's just my opinion.  


![image](https://github.com/hashicorp/vault/assets/7660718/850d3906-7d61-4a8c-bbde-b5b312177b24)

Reply:
I want to create a simpler diagram to include, but figured I'd get the doc reviewed in the meantime :)

Reply:
I also agree, but it will have to be a fast follow on Friday/Monday. I won't be able to clean up the image and finish the other PR reviews before I leave for the team offsite this evening.